src/cdk/portal/portal.spec.ts

```diff
@@ -299,15 +299,15 @@ describe('Portals', () => {
       fixture.detectChanges();
 
       // Attach the TemplatePortal.
-      testAppComponent.portalWithBinding.attach(host);
+      testAppComponent.portalWithBinding.attach(host, { $implicit: { status: 'fresh' } });
```

Comment:
There should be:
* One test where no context is given
* One test where context is given in the constructor
* One test where context is given in `attach`
* One test where context is given in both the constructor and in attach